### PR TITLE
Make the default XPath version implementation defined

### PR DESCRIPTION
**Pull request by ndw**
Closes issue 39 in the vnext repository.

### CODE REVIEW

File: xproc/src/main/xml/specification.xml [hunk 1]
```diff
@@ -6352,10 +6352,17 @@ If the attribute is not specified, the value
...
-assumed. <error code="S0110">It is a <glossterm>static error</glossterm> if
+to the constraints outlined in <xref linkend="xpath-context"/>.
+<error code="S0110">It is a <glossterm>static error</glossterm> if
 the requested XPath version is less than “<literal>3.1</literal>”</error>.</para>
+<para><impl>If a pipeline does not request a specific XPath version, the version
+used is <glossterm>implementation-defined</glossterm>.</impl> <impl>If different
+pipelines or libraries declare different XPath versions, it is
+<glossterm>implementation-defined</glossterm> how those conflicts are resolved.</impl>
+An implementation might use different versions for different pipelines, or it might
+use the same version for all pipelines. <impl>If an implementation elects to us the
```
Comment (gimsieke) [inline xproc/src/main/xml/specification.xml:6363]: us → use

**Review by gimsieke — CHANGES_REQUESTED** [review]
minor typo

**Comment by xml-project** [comment]
I think this is a good solution for the inheritance problem. But I am not sure how it covers the problem when the pipeline is asking for an XPath version not supported by the processor. Does this PR suggest the reaction as implementation-defined?

**Comment by ndw** [comment]
Fixed the typo. Extended `err:XS0110` to cover versions less than 3.1 or other versions unsupported by the processor.

**Review by gimsieke — APPROVED** [review]

**Pull request merged**